docs/apm/using-the-apm-ui.asciidoc

```diff
@@ -2,90 +2,295 @@
 [[apm-ui]]
 == Using the APM UI
 
-The APM UI is designed to be as intuitive as possible, but you might come across certain terms or concepts that don’t feel native to you. Not to worry, that’s why we’ve created this guide to help you get the most out of the solution.
+The APM UI is designed to be as intuitive as possible,
+but you might come across certain terms or concepts that don’t feel native to you.
+Not to worry, we've created this guide to help you get the most out of Elastic APM.
 
 * <<services>>
+* <<traces>>
 * <<transactions>>
-* <<spans>>
+** <<transaction-details>>
+** <<spans>>
 * <<errors>>
-* <<errors-alerts-with-watcher>>
+** <<errors-alerts-with-watcher>>
 * <<machine-learning-integration>>
+* <<metrics>>
 * <<query-bar>>
 
 [[services]]
 === Services
 
-You can add services by setting the `service.name` in the APM agent configuration per service you’re instrumenting. The global time range filter in Kibana defines which services are available. The Services overview gives you a quick insight into the health and general performance of each service.
+The Services overview page gives you quick insights into the health and general performance of each service.
+The <<set-time-filter,global time range filter>> in {kib} defines which services are available.
+
+You can add services by setting the `service.name` configuration in each of the {apm-agents-ref}[APM agents] you’re instrumenting.
 
 [role="screenshot"]
 image::apm/images/apm-services-overview.png[Example view of services table the APM UI in Kibana]
 
+[[traces]]
```

Comment:
Added a section on the Traces page and DT

Reply:
Great to get this in 👍